addons/hr_timesheet/tests/test_timesheet.py

```diff
@@ -616,7 +616,7 @@ def test_uom_change_timesheet(self):
             'employee_id': self.empl_employee2.id
         })
         self.env.company.timesheet_encode_uom_id = self.env.ref('uom.product_uom_day')
-        self.assertEqual(project.total_timesheet_time, 8, "Total timesheet time should be 8 hours")
+        self.assertEqual(project.total_timesheet_time, 1, "Total timesheet time should be 1 day")
```

Comment:
This change seems to contradict the test description? Are we changing the default from hours to days or something? (didn't inspect very carefully, maybe just the description needs to be updated)